Makefile

```diff
@@ -80,9 +80,16 @@ analyze: python
 	@tox -qe analyze
 
 .PHONY: format
-format: python
+format: backend-format frontend-format
```

Comment:
It might be nice to have these as `format-backend` and `format-frontend`. That way you can tab complete up to `format` and then get the option of the three as tab completion options.

Reply:
I agree. I tried to be consistent with other targets in  `lms` and  in `h`:

`backend-tests` <-> `frontend-tests`
`backend-lint` <-> `frontent-lint`

We can change these on a follow up PR.

Reply:
Maybe the original idea was to type `backend` and tab autocomplete to get all the backend options.